### ISSUE TITLE
Feature Request: Optional grayscaled tray-icon

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#51]
```
Another feature request - i hope that's ok.

Right now the autokey-tray-icon is blue/white colored and is an eye-catcher if 
almost all other tray-icons (i.e. in gnome) are grey or greyscaled.

Would be a nice eye-candy feature if the user could select between a colored & 
grey icon set in autokeys preferences.



```

Original issue reported on code.google.com by `elAndroi...@googlemail.com` on 7 Jul 2010 at 5:45

**Comment by GoogleCodeExporter** [comment]
Original comment by `cdekter` on 9 Jul 2010 at 1:35
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in v0.70.5
```

Original comment by `cdekter` on 14 Jul 2010 at 3:40
- Changed state: **Fixed**
